tests/test_document_collection.py

```diff
@@ -134,6 +135,42 @@ def test_create_and_run_process(document_collection, requests_mock):
     assert_process_equal(expected_process, actual_process)
 
 
+def test_create_process(document_collection, requests_mock):
+    process_name = "test-process"
+    state = "IDLE"
+    number_of_documents = 12
+
+    requests_mock.post(
+        f"{API_EXPERIMENTAL}/textanalysis/projects/{PROJECT_NAME}/processes",
+        headers={"Content-Type": "application/json"},
+        json={"payload": None, "errorMessages": []},
+    )
+
+    requests_mock.get(
+        f"{API_EXPERIMENTAL}/textanalysis/projects/{PROJECT_NAME}/"
+        f"documentSources/{document_collection.name}/processes/{process_name}",
+        headers={"Content-Type": "application/json"},
+        json={
+            "payload": {
+                "processName": process_name,
```

Comment:
Here another reason for not having the process type in the Python client becomes visible: I believe we cannot query the process type from the product atm. We can only query if it has a pipeline or not.